examples/examples-cpu/.saturn/start

```diff
@@ -1,4 +0,0 @@
-pip install \
```

Comment:
since `start` isn't required, can you please just remove this file instead of leaving a blank one here?

Reply:
sure thing!